sponsorship_compassion/models/res_partner.py

```diff
@@ -98,6 +99,28 @@ def _compute_count_items(self):
                 ('partner_id', '=', partner.id),
                 ('account_id.code', '=', '1050')])
 
+    @api.multi
+    def set_privacy_statement(self, origin):
+        for partner in self:
+            if partner.signed_privacy_statement:

```

Comment:
`partner.signed_privacy_statement` is not defined. You should add this field I guess or do something else.